pkg/services/sqlstore/quota.go

```diff
@@ -155,7 +155,7 @@ func (ss *SQLStore) GetUserQuotaByTarget(ctx context.Context, query *models.GetU
 		var used int64
 		if query.Target != alertRuleTarget || query.UnifiedAlertingEnabled {
```

Comment:
wondering if we should also change this to query.Target to ensure we are looking at the users table. Someone could target some other table for `user_ids`

Reply:
@papagian could you comment on this query.Target, do you know what the query.Target is targeting other than user table?

Reply:
this is solved by adding a notServiceAccount function to each target.Query that targets the "user" table